lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -46,6 +47,13 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     }
   }
 
+  function preventDefaultWhenModifierKeyIsNotPressed(evt: KeyboardEvent) {
```

Comment:
This functionality should probably be moved to a utility.  In this PR we have two places that need basically the same logic, and there are or will be other places in RSC that will as well as we implement more advanced keynav.

Reply:
> This functionality should probably be moved to a utility. In this PR we have two places that need basically the same logic, and there are or will be other places in RSC that will as well as we implement more advanced keynav.

Thanks, I moved it to it's own file here: 56973dc6a829f285e56a1ce4295a8c7d3221040c